doc/manual/de/config/xml-format.rst

```diff
@@ -34,6 +34,14 @@ In der zweiten Zeile sind folgende Einstellungen relevant:
 +----------------------------+--------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------+------------+
 | ``screensave_page="main"`` | Mit dieser Option kann festgelegt werden, auf welche Seite nach Ablauf von ``sceensave_time`` angezeigt wird | Angabe der Seite-ID zB. "id_1" bzw. Seitenname zB. "Main"                  | NEIN       |
 +----------------------------+--------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------+------------+
+| ``backend="mqtt"``         | Mit dieser Option kann das Backend festgelegt werden und die vom Server gesendete Information übersteuern    | knxd, oh, mqtt                                                             | NEIN       |
++----------------------------+--------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------+------------+
+| ``backend-url="ws://..."`` | Nur für MQTT: URL der WebSocket Schnittstelle des MQTT Brokers                                               |                                                                            | NEIN       |
++----------------------------+--------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------+------------+
+| ``username="user"``        | Nur für MQTT: Benutzername des MQTT Brokers                                                                  |                                                                            | NEIN       |
++----------------------------+--------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------+------------+
```

Comment:
openHAB supports (and or the CometVisu-openHAB plugin even needs) login with credentials. So this it not only for MQTT